### DISCUSSION TITLE
¿Qué ocurre con el fichero que ya tuvieramos de .zshrc?

### DISCUSSION THREAD

**Discussion by Bizkaitarra** [CodelyTV/dotly#152] [Q&A]
Tenía un fichero .zshrc con cosas mías y al instalar dotly lo he perdido, sin advertencia al respecto.

¿Se queda guardado en algún lado?

Por suerte tengo una copia de la mayoría de la cosas pero me toca recordar algunas otras.

**Comment by gtrabanco** [comment]
No se guarda. Hay un issue abierto al respecto #76

Trato de resolver esto en una PR abierta pero es un borrador aún.

**Comment by Bizkaitarra** [comment]
Muchas gracias. Por suerte he recuperado ya el fichero pero sería
interesante al menos hasta sacar la PR incluir la advertencia en el readme.

No me queda muy claro si cuando actualice dotly me volverá a quitar lo que
tengo metido en el zshrc o si hay que meter las cosas propias de otra forma

Un saludo

**Reply by gtrabanco** [reply]
Para meter las cosas personalizadas usa los ficheros en la carpeta `shell` que se llaman `aliases.sh`, `exports.sh` y `functions.sh`.

Estos archivos se precargan a través del archivo `init.sh`.

Si necesitas actualizar la variable `PATH` con nuevos valores, utiliza el array `path` en el archivo `exports.sh`. Cuando se haga merge del PR #129 esto cambiará ligeramente y el array `path` pasara a estar en un archivo `paths.sh`. También se actualizará el contenido del archivo `.bashrc` y `.zshrc`. Atento cuando eso suceda.

Si que estaría bien un aviso en el README antes de los pasos de instalación sobre la sobreescritura de estos archivos @rgomezcasas, es importante.

**Reply by rgomezcasas** [reply]
Totalmente, está en el backlog!! 😊
